### PR TITLE
added dbi

### PR DESCRIPTION
**Pull request by JonasPfeifer05**

### CODE REVIEW

**Review by oliverpk2000 — COMMENTED** [review]
Lgtm

File: DBI/Speicher Und Buffer/Maintenance Effort.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Der Aufwand, um einen Speicher sicher am Laufen zu halten. 
+
```
Comment (ToBinio) [inline DBI/Speicher Und Buffer/Maintenance Effort.md:6]: could/should be expanded with some examples similar to "Zugriffszeit"

File: DBI/Speicher Und Buffer/DBMS Architektur.md
Comment (ToBinio) [inline DBI/Speicher Und Buffer/DBMS Architektur.md:1]: ***principle of locality***!!!

**Review by ToBinio — CHANGES_REQUESTED** [review]

File: DBI/ISAM/Vorlage.md
Comment (oliverpk2000) [inline DBI/ISAM/Vorlage.md:1]: ? should this be here?

File: DBI/Speicher Und Buffer/DBMS Architektur.md
Comment (oliverpk2000) [inline DBI/Speicher Und Buffer/DBMS Architektur.md:1]: feels like the wrong folder for this, but we can move this later

**Pull request merged**